static/src/javascripts/projects/common/modules/email/run-checks.js

```diff
@@ -194,6 +204,7 @@ define([
         getEmailInserted: getEmailInserted,
         allEmailCanRun: allEmailCanRun,
         getUserEmailSubscriptions: getUserEmailSubscriptions,
-        listCanRun: listCanRun
+        listCanRun: listCanRun,
+        checkTestClash: checkTestClash
```

Comment:
Yes kind of.  Its actually used in userIsInAClashingAbTest line 45.  Maybe I should rename it to remove the word test although its only exposed outside this file so I can call it in the unit test.